demo/src/main/java/com/example/demo/model/User.java

```diff
@@ -0,0 +1,57 @@
+package com.example.demo.model;
+
+import javax.persistence.*;
+
+@Entity
+public class User {
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private Long id;
+    @Column
+    private String firstName;
+    @Column
+    private String lastName;
+    @Column
+    private int age;
+
+    public User() {}
+    public User(String firstName, String lastName, int age) {
```

Comment:
I recommend you remove super() constructor and follow the same argument sequence as you stated in the User object (id, firstName, lastName)

Reply:
change getters in setters accorfingly in the proper order